js/components/utils.js

```diff
@@ -165,6 +165,94 @@ var Utils = {
   },
 
   /**
+   * Convenience function for base64 decoding.
+   *
+   * @function decode64
+   * @param {String} s The base64 encoded string to decode.
+   * @return {String} The String representing the decoded base64.
+   */
+  decode64: function(s) {
+    s = s.replace(/\n/g,"");
+    var results = "";
+    var j, i = 0;
+    var enc = [];
+    var b64 = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
+
+    //shortcut for browsers with atob
+    if (window.atob) {
+      return atob(s);
+    }
+
+    do {
+      for (j = 0; j < 4; j++) {
+        enc[j] = b64.indexOf(s.charAt(i++));
+      }
+      results += String.fromCharCode((enc[0] << 2) | (enc[1] >> 4),
+                                      enc[2] == 64 ? 0 : ((enc[1] & 15) << 4) | (enc[2] >> 2),
+                                      enc[3] == 64 ? 0 : ((enc[2] & 3) << 6) | enc[3]);
+    } while (i < s.length);
+
+    //trim tailing null characters
+    return results.replace(/\0/g, "");
+  },
```

Comment:
https://caniuse.com/#search=atob
https://help.ooyala.com/video-platform/concepts/pbv4_environments.html
Seems like `atob` exists in all officially supported by Ooyala browsers. So custom implementation is redundant.

Reply:
@zelibobla even so, I think it is worth it to include this helper to have as broad support as possible. Customers could report issues if their user base is watching with older or unsupported browsers and this is a cheap defense against that